src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
 }
 
 /**
- An enumeration that describes the allele that corresponds to a genotype.
+ Allele encodings for genotypes.
  */
-enum GenotypeAllele {
+enum Allele {
```

Comment:
Is it possible (or even realistic) to get rid of enums? Impala doesn't support importing our parquet formats because of them. I doubt this is what we want, but I wanted to open a discussion about it.